pkg/schedule/schedulers/metrics.go

```diff
@@ -116,15 +116,15 @@ var (
 			Subsystem: "scheduler",
 			Name:      "store_slow_trend_action_status",
 			Help:      "Store trend scheduler calculating actions",
-		}, []string{"reason"})
+		}, []string{"type", "status"})
 
 	storeSlowTrendMiscGauge = prometheus.NewGaugeVec(
 		prometheus.GaugeOpts{
 			Namespace: "pd",
 			Subsystem: "scheduler",
 			Name:      "store_slow_trend_misc",
-			Help:      "Store trend internal uncatalogued values",
-		}, []string{"type"})
+			Help:      "Store trend internal uncatelogued values",
```

Comment:
Should it be `uncatalogued`?
https://dictionary.cambridge.org/zhs/%E8%AF%8D%E5%85%B8/%E8%8B%B1%E8%AF%AD/uncatalogued

Reply:
Yep, it's a spelling error. I'll tidy and clean up all misleading metrics in the next pr.